test/test_jit.py

```diff
@@ -3805,6 +3805,25 @@ def func(x, value1, value2):
         check_dynamic_indexing("[i + j]", consec((3, 3)), 0, 1)
         check_dynamic_indexing("[i:j, i]", consec((3, 3, 2)), 0, 2)
 
+    def test_index_ellipses(self):
+        vals = [":", 1, None]
+        for _ in range(100):
+            indices = [random.choice(vals) for _ in range(4)]
+            indices[random.randint(0, len(indices) - 1)] = "..."
+            test_str = dedent("""
+            def f():
```

Comment:
I suppose we could, but we use `execWrapper` everywhere else in our tests as well, and it would require us to wrap it in a module.

Is there any particular reason to prefer `ScriptModule.define()`?